### DISCUSSION TITLE
COFFEE PROPOSAL

### DISCUSSION THREAD

**Discussion by JcEii** [sunflower-land/sunflower-land#1880] [Ideas]
### Dcol and I would like to propose the introduction of coffee in Sunflower Isles!

The aim is to introduce a new food item (beverage technically) to be used for XP or part of future requirements/events. 

Ingredients: Coffee bean and milk 
Output: Coffee cup = 500xp and 5-10% chance of fertiliser spawning 

![sample2](https://github.com/sunflower-land/sunflower-land/assets/105348036/8c0487ac-1291-4b4f-acd9-313cbb7b8fe1)

The overall process of creating brewed coffee would be similar to the creation of cakes, where ingredients are made from 2 different mechanics: Crops coming from ‘Farming’ and eggs coming from ‘Animals’. Just like eggs and crops created in the bakery, coffee bean and milk will be created in the cafe’, a new building to be introduced. 
This would add 1 utility for the eventual ‘Cow’ animal once it is ready.

When collecting a coffee cup from the café’ there is a 5-10% chance of a fertiliser potion appearing. This is tied to the leftover grinded coffee which in real life acts as a great fertiliser for plants!

### PROCESS:
Cows -> Milk 
Coffee Seed -> Coffee Bean -> "Cafe’", Coffee Bean + Milk -> Coffee Cup 

### ART:
**Café:**
_Sample:_
![final_coffee_stall_big](https://github.com/sunflower-land/sunflower-land/assets/105348036/1937b0bf-8d4e-456f-8276-220c401c337a) 
![final_coffee_stall](https://github.com/sunflower-land/sunflower-land/assets/105348036/46cb806b-f6ed-422a-808c-e8a7b66d58ec) 
![final_coffee_stall_2](https://github.com/sunflower-land/sunflower-land/assets/105348036/f4ba59ff-c03b-4411-bc37-e9e0153b2df1)

_Minified:_
![final_coffee_stall](https://github.com/sunflower-land/sunflower-land/assets/105348036/a7f87510-1e63-4c1e-8a34-aa12e95660a0)  
![barista](https://github.com/sunflower-land/sunflower-land/assets/105348036/66874090-5a3b-452a-82ac-1160e4be2167) 
![barista2](https://github.com/sunflower-land/sunflower-land/assets/105348036/c031dfd4-7350-4a21-b346-8e5d51112a9e)

**Coffee patch:**
_Sample:_
![coffee_patch_big](https://github.com/sunflower-land/sunflower-land/assets/105348036/76e3d90c-70b9-40e8-b257-956bbef7a8db)

_Minified:_
![coffee_patch](https://github.com/sunflower-land/sunflower-land/assets/105348036/3a6d18b4-1348-422e-8925-5b719d935167)

**Coffee Seed and Crop:**
_Fruit Sample:_
![fruit](https://github.com/sunflower-land/sunflower-land/assets/105348036/624effbc-240c-424e-b255-a16e4b0efd5e) 

_Fruit Minified:_
![fruit1](https://github.com/sunflower-land/sunflower-land/assets/105348036/a501aaeb-1a7a-4a75-94fe-8e1696d61377) ![fruit2](https://github.com/sunflower-land/sunflower-land/assets/105348036/9122288f-fb8d-476f-89c6-b6526d193686) ![fruit3](https://github.com/sunflower-land/sunflower-land/assets/105348036/b9ed4ba2-e758-4cac-9c4c-c0cf1db6a87f) ![fruit4](https://github.com/sunflower-land/sunflower-land/assets/105348036/48d40c32-31f5-4b03-a6de-6faada13ed5e) ![fruit5](https://github.com/sunflower-land/sunflower-land/assets/105348036/8a41201b-465c-4ef2-a2a1-b5da31fa33ee) ![fruit6](https://github.com/sunflower-land/sunflower-land/assets/105348036/49af67c5-c225-4fb5-a67d-30ddc9e914e7) ![fruit7](https://github.com/sunflower-land/sunflower-land/assets/105348036/321a4e2b-f0f2-4759-bb53-583ee2cfaf77) ![fruit8](https://github.com/sunflower-land/sunflower-land/assets/105348036/0cb02514-c8a0-4333-81b1-2cbb4933e200) ![fruit9](https://github.com/sunflower-land/sunflower-land/assets/105348036/53ae65ad-6db6-4021-a74a-3844c305cdfe) ![fruit10](https://github.com/sunflower-land/sunflower-land/assets/105348036/29634138-c55c-462e-8cf8-d2000a73c315)

_Seed Sample:_
![seed](https://github.com/sunflower-land/sunflower-land/assets/105348036/d5b9b783-cd9d-4a9f-8a1a-d0e3586d6d20) 

_Seed Minified:_
![seed1](https://github.com/sunflower-land/sunflower-land/assets/105348036/09c80cb5-d29e-477f-b46a-5cbfc2085631) ![seed2](https://github.com/sunflower-land/sunflower-land/assets/105348036/702b19b9-ff17-47ce-a297-5982176c6614)

**Coffee cup:**
_Sample:_

![coffee](https://github.com/sunflower-land/sunflower-land/assets/105348036/bf38196c-715c-421d-9931-d42009c45023) 

_Minified:_
![coffee1](https://github.com/sunflower-land/sunflower-land/assets/105348036/8e9c2cc9-1513-4de4-a0e2-2834ec9517d0) ![coffee2](https://github.com/sunflower-land/sunflower-land/assets/105348036/93cdc65f-7ebe-4ade-a1ab-5917b7e2832f) ![coffee3](https://github.com/sunflower-land/sunflower-land/assets/105348036/61308c59-fe76-42b5-8277-4a304db4d268) ![coffee4](https://github.com/sunflower-land/sunflower-land/assets/105348036/eaaec9b6-6791-4b47-a1eb-fc2d13edf4ea)

We believe by having coffee in the game by combining Farming and Animal products will provide great gameplay loops, specialisations, market stimulus and progression for players. NFTS that boost production in milk can be thought of and considered, along with nfts that increase fertiliser spawn chance and increase coffee crop and cup yield.

"We know that coffee fruit doesnt magically turn into coffee beans and there is a process behind it, but this is sunflower land where wheat also turns magically into flour for cakes and goblins exist 😁"

**Comment by mmaxxpeinn** [comment]
The stall with coffee maker is awesome 🤩

I absolutely love this idea! Can't farm in the morning without coffee, so we should have this!

**Comment by DaanMich** [comment]
Love the idea! 
Personally I’d prefer the second art since it’s  super clear that it’s a coffee stand 🤗

Can’t wait for coffee in the game LFG

**Comment by LeShykun** [comment]
Great proposal and artwork love if 😍, can you provide some pictures of this building inside the game? Just to see how it would look like next to some existing buildings! ❤️

**Comment by johannesunana** [comment]
Amazing idea! Let's hope the developers and the community consider this in the coming weeks/months. ✨

**Comment by godlevel12** [comment]
I love the idea, I love that there is a process. Cant wait to see it in game.
I liked the second building more ❤️

**Comment by stevew00dy** [comment]
I think when you consider Coffee here you also need to consider the game loop in the sense of - Are we making this mythical or more like reality. 

Are there other buildings involved like Fermenting, Roasting etc or is it straight from seed to cup. 

Depending on the direction that the developers feel is best, this can remain a simple core game loop or it can become an more complex one for the hardcore players. 

Might be worth considering what the very simple process as well as the complex process would look like and see how the community overall feel with some voting. 

![image](https://user-images.githubusercontent.com/5225886/208008235-5f17bd4e-5877-4ea0-8624-beff3ceca1b0.png)

**Reply by TheDcol** [reply]
![Screenshot_20221216-181259](https://user-images.githubusercontent.com/108110065/208043258-1c296e21-a839-4e54-b90c-398e7d70cae4.png)

Ye we stated in the discussion it would be 'magical' like the cakes

**Comment by Polysemouse** [comment]
I  the proposal and the simplicity of creating coffee from beans and milk directly.  The bonus fertilizers would be an interesting addition to the gameplay.

Each crop in the game has 4 stages of growth stages (4 different PNGs).  For the art of the growing coffee bean, it would be nice to see more variations between the 4 growth stages (the current ones look too similar to each other).  Also the seed size can be increased to match with the ones existing in game.

**Comment by cryptosmikey** [comment]
What if there were 2 loop styles; Simple loop where you get basic seed to coffee. Dynamic loop you get the full out process where you can create different kinds of coffee grind blends and drinks hot & cold, latte's, espresso shots, decaf, regular, medium, dark roast etc. Coffee is my passion so hit me up if you have any questions.

**Comment by TheDcol** [comment]
wen

**Comment by JcEii** [comment]
WEEEENNNNN

**Comment by MoonbaggerA** [comment]
abit late, but great idea!

**Comment by Jericho-rpc** [comment]
wen
